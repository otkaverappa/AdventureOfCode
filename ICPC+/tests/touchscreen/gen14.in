19
padeplzrhxocwo 10
ztloncbklwrlqx
oxzgxckmnlgxiu
zglycyopapamot
zesqbswywimmts
jdpilajnaxdbpf
pdyqhtgileijmu
vhnemepyeylgxq
jmvarwrapziags
junhcclqgcomac
uaseooegkxgzzr
bfjmzyvdchukli 10
yoktodzctfkddr
eekhjvviurmyag
llqjzbepgdsbjc
eovksftbbolxhx
xhfivwswzwmhce
iliozearltumjf
lrfjakrxilujkg
qomazwoaaotmjn
atuomzxomqowei
gqqyeezdanfdey
rnnrihfwhenvxb 10
rbjxrbwvfvafkh
iofzestobalkfy
fczxfkuwnstsnv
zaciqjjudcjedw
pkwuoxrthopugk
owhnwjvmuhjxju
doquamrojkhsyy
ohicdprzapnuwy
thszwkvyymnjxu
bxtredvhvnivcw
syunfpmdzicywr 10
htlltgczmahhnr
cppwpmkubyacge
cdvkyixtpbtdbc
kotofjluxvrytt
ccxfhsphcobtpw
wtygjtvpeglbdc
zyvccuhjmytpow
kdshysnilkxqsk
rvntwkxyfgktgf
kubvzvcynriach
surkseforeowma 10
rufcqixpdbpssz
tuglpyxheexvkl
twonstpieoyjpp
cjpxexkvvkecqb
zbotzcgsyxaena
nespnjotgzqelu
gbxfcoacsiwshw
wuzkyrbmashity
meusfsaigamyli
tsgpnfblyfzaxh
jqhvubqzvqjdsx 10
edhxwomlvqwuvy
ushdkobertfnjr
sdowjwtfmiqhyo
bvmypvdzjefdzm
skdkouhxqceekw
lkkpfznuusvgza
jypdjunzowwgza
klwwyinfhcceux
ktavupydjlcaka
gjcrwapukcasec
yabiubdqsdveoz 10
eyzkjddgdscnue
hagfbipxlungxi
kojooiaynegqyk
fvpnvxuyhmvvgk
beuludbindiajq
sidydsnarhbztw
ucjvidichlmwox
xynpgqplkcncmo
dfnzjwwrbgvkri
hffgduvlnlyzno
dzeghrhspdkrjh 10
bapiiwqntmbgzz
hoplqutzlbrded
uqlxsaiazyoskr
alsjaiuscorppi
svmolxmgzugasw
sdpuqieqqaksob
jfmcbyqoxcwywd
zrbtwromzufrup
llruqfytdrkctg
aslzlpthijtjfa
abqnojhhogatxm 10
vsvxmixxxsghda
qiaslsfadphrxj
kuwhotfdbecbyj
iblbjnvwgbwjsd
dpnpllzzegefkh
gisrlduwqpsysp
kksnzheksdmwjq
cvzkfrdquxmnnh
lfwvsqkrypeqss
neirzkdhbhxygm
lttwbruvheohts 10
yoknuvetfjcisa
gzorujqvaksjoh
tkzramgujkpqvs
yoshniyhroftaz
epgxbiqbuwvfik
veevuychgdqztx
uuxzlfymnpnjnk
owwlcbhwzlfiox
hkxcewdrbbdpss
ygcpeaagbjcdvj
llguxdydzbudfx 10
szrshvhlwktztx
eqgrepmbskfuoz
xvzruqjcotpkdj
lygqrnhvcvywgf
swhrrgilyupopf
auomsweljlgnjh
mrmentwebgrccj
stotnefiajtmwc
zfjlwxsltqruwl
yburwimkoruocn
aapcganeyfsrvj 10
nuuoxoftyrfmkz
donfoehwfxbdet
waejwzxvpeqoxw
chxhxknmruiyrl
bwgxyliwkgtbkj
pkfutdbrpqdiln
gezjcfiaqrycxr
ejbvvirpntgfjl
nwyubxdddmgvdg
ycycnbzljraxmi
extrutlxrrcwfi 10
tkrropvdqvoboo
aayextwtnjqgct
ehdxturhlmlbjz
fzqfzqkzkisxtj
fyejfhizezipot
rzvyzndadpznzu
mtdtthcyqnzumk
kcfdcabdpedtwf
ivzxqeqjnvkdil
yuvkzbnbdqgtwj
osqyqrvivouije 10
orronnymqnptdv
ozhcuxbkrysmmm
xvsnokbczbopoe
krzzsidmggwxfr
ltfkpyzdkahjby
yrckidjbmnputm
tafguktlisqsty
dwwdoaoyfzbtmr
pifikkreukrfei
xzgbwegmgukluo
fifwsmfdwyjqkb 10
volvotymxeydzk
qvaxdhuwtbbpzm
gjpdzaantybsfb
xeoobolewhcpif
hjspujsukvjftl
yyoveclhtzmpgq
gpwpyqetzzpluy
rplrqaoueaexzq
ofixwgmxyrsash
lmhedtvtvmqbou
zpmpwumtdzqbsi 10
bkrpyytbtrvqdl
sthritgepvxsup
wmzxyqmwrixkzs
beftxmmhhulxpl
plclxelxuztlhr
yilzotunhiupei
muvdhxrhbceydy
lnpjxckmvgzfqw
uvegrblyycfcgk
akinxzzwdlibsk
giiafoiwptvnwc 10
rcmtowhlxgkbrs
clekwmkbasapnv
fjzyomscizqiha
jaunozxkljnodn
fqkkckjqybsick
qlkzlfozgllrwz
hzonsbxuniknle
vprmabnoienorb
iocrnseftebimn
vautrlfsouizyv
pqyzgarvtvdoze 10
xosuqmnhzubnrj
mphehfdpgulbso
prsohmjzayhzti
pmudbbhlglaohn
pzbhtwxckiclhj
mctbonhrqoezbh
nlufkyofulieuk
rdvefqhvgomwfr
vgaltuseuglott
vnfmqasxraszqh
vvateaexwwdrep 10
fxiclordqjdjlx
icegxezdgfbdbe
uftbeegpuxvmia
wwxgyeoxknbrse
uviqcethkzyeyv
qhvofvxdzlcjzf
atjwruovaiekjf
qicgraxwvubufd
fglicwgtsxqsfv
eqcuyedreanzwo
