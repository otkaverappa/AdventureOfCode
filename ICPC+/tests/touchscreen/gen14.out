uaseooegkxgzzr 29
pdyqhtgileijmu 48
jmvarwrapziags 61
oxzgxckmnlgxiu 62
junhcclqgcomac 63
vhnemepyeylgxq 66
jdpilajnaxdbpf 68
zesqbswywimmts 68
ztloncbklwrlqx 77
zglycyopapamot 92
lrfjakrxilujkg 34
iliozearltumjf 43
xhfivwswzwmhce 46
gqqyeezdanfdey 47
qomazwoaaotmjn 50
eekhjvviurmyag 52
eovksftbbolxhx 54
yoktodzctfkddr 55
llqjzbepgdsbjc 57
atuomzxomqowei 60
fczxfkuwnstsnv 38
thszwkvyymnjxu 40
rbjxrbwvfvafkh 41
bxtredvhvnivcw 42
ohicdprzapnuwy 50
owhnwjvmuhjxju 51
zaciqjjudcjedw 51
pkwuoxrthopugk 53
doquamrojkhsyy 55
iofzestobalkfy 55
kotofjluxvrytt 42
zyvccuhjmytpow 46
wtygjtvpeglbdc 47
rvntwkxyfgktgf 48
ccxfhsphcobtpw 52
htlltgczmahhnr 52
cppwpmkubyacge 53
kdshysnilkxqsk 58
cdvkyixtpbtdbc 59
kubvzvcynriach 61
rufcqixpdbpssz 36
wuzkyrbmashity 44
zbotzcgsyxaena 44
tuglpyxheexvkl 50
meusfsaigamyli 51
tsgpnfblyfzaxh 51
gbxfcoacsiwshw 54
twonstpieoyjpp 58
nespnjotgzqelu 60
cjpxexkvvkecqb 62
bvmypvdzjefdzm 37
jypdjunzowwgza 44
ushdkobertfnjr 45
ktavupydjlcaka 53
klwwyinfhcceux 55
skdkouhxqceekw 57
edhxwomlvqwuvy 58
lkkpfznuusvgza 60
gjcrwapukcasec 61
sdowjwtfmiqhyo 64
eyzkjddgdscnue 37
ucjvidichlmwox 42
beuludbindiajq 44
sidydsnarhbztw 50
kojooiaynegqyk 51
dfnzjwwrbgvkri 54
fvpnvxuyhmvvgk 61
hagfbipxlungxi 62
hffgduvlnlyzno 64
xynpgqplkcncmo 66
llruqfytdrkctg 48
sdpuqieqqaksob 49
aslzlpthijtjfa 53
uqlxsaiazyoskr 54
zrbtwromzufrup 54
hoplqutzlbrded 57
alsjaiuscorppi 60
bapiiwqntmbgzz 60
svmolxmgzugasw 60
jfmcbyqoxcwywd 64
kuwhotfdbecbyj 42
qiaslsfadphrxj 45
iblbjnvwgbwjsd 48
gisrlduwqpsysp 49
neirzkdhbhxygm 49
cvzkfrdquxmnnh 50
lfwvsqkrypeqss 56
dpnpllzzegefkh 57
vsvxmixxxsghda 60
kksnzheksdmwjq 64
uuxzlfymnpnjnk 46
veevuychgdqztx 46
epgxbiqbuwvfik 47
hkxcewdrbbdpss 50
yoshniyhroftaz 50
tkzramgujkpqvs 52
owwlcbhwzlfiox 53
yoknuvetfjcisa 54
gzorujqvaksjoh 58
ygcpeaagbjcdvj 60
lygqrnhvcvywgf 31
mrmentwebgrccj 45
stotnefiajtmwc 48
szrshvhlwktztx 52
auomsweljlgnjh 57
yburwimkoruocn 57
eqgrepmbskfuoz 58
xvzruqjcotpkdj 62
swhrrgilyupopf 64
zfjlwxsltqruwl 65
nwyubxdddmgvdg 43
bwgxyliwkgtbkj 45
ycycnbzljraxmi 45
donfoehwfxbdet 47
waejwzxvpeqoxw 47
ejbvvirpntgfjl 52
gezjcfiaqrycxr 56
pkfutdbrpqdiln 56
chxhxknmruiyrl 58
nuuoxoftyrfmkz 62
fzqfzqkzkisxtj 39
rzvyzndadpznzu 47
aayextwtnjqgct 48
kcfdcabdpedtwf 48
tkrropvdqvoboo 50
mtdtthcyqnzumk 51
yuvkzbnbdqgtwj 51
fyejfhizezipot 53
ivzxqeqjnvkdil 61
ehdxturhlmlbjz 62
xzgbwegmgukluo 41
ozhcuxbkrysmmm 46
krzzsidmggwxfr 49
orronnymqnptdv 50
yrckidjbmnputm 52
xvsnokbczbopoe 53
dwwdoaoyfzbtmr 54
ltfkpyzdkahjby 56
pifikkreukrfei 59
tafguktlisqsty 60
gjpdzaantybsfb 40
lmhedtvtvmqbou 46
ofixwgmxyrsash 48
hjspujsukvjftl 52
qvaxdhuwtbbpzm 52
volvotymxeydzk 53
xeoobolewhcpif 53
yyoveclhtzmpgq 58
rplrqaoueaexzq 59
gpwpyqetzzpluy 60
bkrpyytbtrvqdl 45
akinxzzwdlibsk 49
muvdhxrhbceydy 52
sthritgepvxsup 54
uvegrblyycfcgk 54
plclxelxuztlhr 55
lnpjxckmvgzfqw 57
beftxmmhhulxpl 60
wmzxyqmwrixkzs 64
yilzotunhiupei 64
vautrlfsouizyv 43
fqkkckjqybsick 45
qlkzlfozgllrwz 47
vprmabnoienorb 50
iocrnseftebimn 57
clekwmkbasapnv 61
hzonsbxuniknle 62
rcmtowhlxgkbrs 62
fjzyomscizqiha 63
jaunozxkljnodn 65
pzbhtwxckiclhj 42
pmudbbhlglaohn 47
rdvefqhvgomwfr 47
vnfmqasxraszqh 56
mphehfdpgulbso 59
mctbonhrqoezbh 60
vgaltuseuglott 60
prsohmjzayhzti 66
nlufkyofulieuk 69
xosuqmnhzubnrj 79
eqcuyedreanzwo 38
fglicwgtsxqsfv 45
icegxezdgfbdbe 46
qicgraxwvubufd 46
wwxgyeoxknbrse 48
qhvofvxdzlcjzf 54
uviqcethkzyeyv 55
atjwruovaiekjf 58
uftbeegpuxvmia 58
fxiclordqjdjlx 62
